frontend/compositions/map/Map.tsx

```diff
@@ -0,0 +1,118 @@
+import * as d3 from "d3"
+import { Feature, Point } from "geojson"
+import { useEffect, useRef, useState } from "react"
+import useResizeObserver from "use-resize-observer"
+import BaseMap from "./BaseMap"
+import { MarkerDescription, MarkerLayer } from "./marker-layer"
+import useData from "./useData"
+import styles from "./map.module.css"
+
+export type BoundingType = [[number, number], [number, number]]
```

Comment:
I think the intention of this declaration can be improved with [narrowing](https://www.typescriptlang.org/docs/handbook/2/narrowing.html): Could we try something like:

```
interface BoundingType {
xStart: number;
yStart: number;
xEnd: number;
yEnd: number;
}
```

Reply:
d3 requires this to be [[number, number], [number, number]] , and any bounds method will return this type, so we'll always encounter it.  It does make sense to have it this way: it is describing a bounding box using two points - upper left and lower right.

Reply:
That's unfortunate, but requirements are requirements. I'd like to look at the docs and see if there's some way to make this more descriptive, but it's not a priority or a blocker.